pkg/occlient/occlient.go

```diff
@@ -167,11 +167,14 @@ func parseImageName(image string) (string, string, string, error) {
 	if len(digestParts) == 2 {
 		// image is references digest
 		return digestParts[0], "", digestParts[1], nil
```

Comment:
validation for empty digestParts also required right?
or only empty digestParts[0] validation needs to be added with safe default for digestParts[1]?

Cc: @surajnarwade @mik-dass